ui/src/views/Scopes/ScopesetExpander/index.jsx

```diff
@@ -59,7 +80,7 @@ export default class ScopesetExpander extends Component {
 
     return (
       <Dashboard
-        title="Expand Scopes"
+        title="Expand Scopesets"
```

Comment:
I am unsure of how to rebase on the master given the fact that I am working on master.

Reply:
You can try doing `git pull git@github.com:taskcluster/taskcluster.git master --rebase`.

Reply:
I tried your mentioned command but was denied because of permissions. However, I did re-run a modified version of the command: `https://github.com/taskcluster/taskcluster.git master --rebase` and it did pull and apply the commits to the `top,` but I'm not seeing a change.

Reply:
Interesting. Maybe being on the "master" branch is not helping. Feel free to close this PR and open a new one on a branch that isn't master. I think that might help.